docs/src/pages/components/skeleton/skeleton.md

```diff
@@ -33,3 +33,7 @@ The component supports 3 variants.
 ## Facebook example
 
 {{"demo": "pages/components/skeleton/Facebook.js", "defaultCodeOpen": false, "bg": true}}
+
+## Delay example
```

Comment:
@mbrookes what about removing the demo? Looking at the experience it provides, isn't it an experience we want to discourage? (and remove the prop too)

Reply:
so remove it all, and just add a delay to the class, like?  

animation: '$animate 1.5s ease-in-out 0.5s infinite animation-delay',

Reply:
> what about removing the demo? (and remove the prop too)

As you wish

Reply:
This is the only study I could find: https://uxdesign.cc/what-you-should-know-about-skeleton-screens-a820c45a571a. It suggests that we could support a wave and pulsating animation. This could be an interesting improvement.

Regarding the delay, I think that we should give up on the demo and new props, to reduce the number of animation on the screen. The new demo personally gives me motion sickness, I don't know about you guys.

Reply:
ill clean the code, and keep the delay, in this pr. 

Then start on a new branch/pr with wave and pulsating animations.